robottelo/ui/hosts.py

```diff
@@ -291,3 +291,9 @@ def fetch_host_parameters(self, name, domain_name, parameters_list):
             self.click(tab_locator)
             result[param_name] = self.wait_until_element(param_locator).text
         return result
+
+    def get_host_property(self, parameter_name):
```

Comment:
Right above this line there's implementation of `fetch_host_parameters`, which will search for host, open it and fetch all the parameters/properties values specified in a list (it uses 'fetch_' prefix instead of your suggested 'property_' though). Please use it instead as this helper is basically a duplicate with less functionality, which also forces you to search and click the host inside the test implementation.